cpp/log/cert.cc

```diff
@@ -860,17 +862,19 @@ Cert::Status Cert::IsValidWildcardRedaction() const {
 
   // If we reach here then the RFC says the CT redaction count extension
   // MUST BE present.
-  X509_EXTENSION* exty;
-  if (GetExtension(NID_ctPrecertificateRedactedLabelCount,
-                   &exty) != Cert::TRUE) {
+  const StatusOr<X509_EXTENSION*> exty(
+      GetExtension(NID_ctPrecertificateRedactedLabelCount));
+  if (!exty.ok()) {
     LOG(WARNING) << "Required CT redaction count extension missing from cert";
```

Comment:
Have we combined the absent / error case here? If so the error message may not be accurate.